cards.tex

```diff
@@ -110,6 +128,8 @@ \section{Frightened}
 \foreach[evaluate=\level as \bonus using int(\level*8)] \level in {1, ..., 10} {
 \Spell{\level}{Heal}
 
+\Traits{\Trait{Healing} \Trait{Necromancy} \Trait{Positive}}
+
```

Comment:
There is not enough space on the "Heal" card for this, so it overflows into a second page :(

Reply:
Yes, we need to do something about the layout and unused whitespace. 

> This likely needs some adjustments before merging, for instance:
>
> - reduce height of the tag row?
> - move tags closer to card title

Btw, the **Steel Shield** card overflows already without tags.